.github/workflows/python-tests.yml

```diff
@@ -48,6 +48,20 @@ jobs:
           # Start Docker service
           sudo systemctl start docker
 
+      - name: Check trader agent image existing
```

Comment:
Thanks for the idea to check this! I believe we can add this check in the Python tests before staring the service [here](https://github.com/valory-xyz/quickstart/blob/main/tests/test_run_service.py#L832).
Also, it would be good to check that trader's image is available for mutiple architectures, as we have missed it in the past once.